connector/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -213,4 +205,19 @@ object Connect {
     .version("3.5.0")
     .intConf
     .createWithDefault(200)
+
+  val CONNECT_ENRICH_ERROR_ENABLED =
+    buildStaticConf("spark.connect.enrichError.enabled")
+      .doc("When true, it enriches errors with full exception messages on the client side.")
+      .version("4.0.0")
+      .booleanConf
+      .createWithDefault(true)
+
+  val CONNECT_SERVER_STACKTRACE_ENABLED =
```

Comment:
Same

Reply:
Same for the performance reason

Reply:
I think it is fine to enable this by default.

Reply:
Done